oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreTest.java

```diff
@@ -413,6 +418,63 @@ public void throttlingWrapperCreatedWhenThrottlingIsEnabled_2() {
     }
     // END -- OAK-9909
 
+    // OAK-9967
+    @Test
+    public void renewClusterIdLeaseTrue() throws IllegalAccessException {
+        DocumentStore documentStore = new MemoryDocumentStore();
+
+        DocumentNodeStoreStatsCollector statsCollector = mock(DocumentNodeStoreStatsCollector.class);
+        doNothing().when(statsCollector).doneLeaseUpdate(anyLong());
+
+        ClusterNodeInfo clusterNodeInfo = mock(ClusterNodeInfo.class);
+        when(clusterNodeInfo.renewLease()).thenReturn(true);
+
+        DocumentNodeStore nodeStore = builderProvider.newBuilder().setNodeStoreStatsCollector(statsCollector)
```

Comment:
Please use the `DocumentMKBuilderProvider` `builderProvider` to create a `DocumentNodeStore`. The `DocumentMKBuilderProvider` rule ensures the `DocumentNodeStore` instance is disposed after the test. Same for all other tests in this PR.

Reply:
@mreutegg I have indeed used the `builderProvider` of `DocumentMKBuilderProvider`.

Reply:
Oops, you are right. Looks good then.